source/shared/core_sqlsrv.h

```diff
@@ -2046,6 +2082,18 @@ namespace core {
 
         r = ::SQLRowCount( stmt->handle(), &rows_affected );
         
+        // On Linux platform
+        // DriverName: libmsodbcsql-13.0.so.0.0
+        // DriverODBCVer: 03.52
+        // DriverVer: 13.00.0000
+        // unixODBC: 2.3.1
+        // r = ::SQLRowCount( stmt->handle(), &rows_affected );  
+        // returns r=-1 for an empty result set. 
+#ifdef __linux__
+        if( r == -1 && rows_affected == -1 )
+           return 0;
```

Comment:
So this block only applies to Linux platform? Also, the version numbers in the comments are outdated